style.css

```diff
@@ -98,6 +98,55 @@
     opacity: 1;
   }
 
+  .modal-pop {
+    display: none;
+    height: 100vh;
+    position: fixed;
+    top: 0;
+    right: 0;
+    left: 0;
+    z-index: 10;
```

Comment:
Kindly update your popup window on mobile screens to match the figma design better🙏

Your design:
<img width="277" alt="image" src="https://github.com/Armynerh/personal_portfolio/assets/37116322/652bb70e-6127-45a4-89f1-a18b2b2ea0ab">

Figma design:
<img width="180" alt="image" src="https://github.com/Armynerh/personal_portfolio/assets/37116322/e238ffdc-f52a-4c57-b3fa-b209cee470b7">

A few things to note:
- Your modal design is full screen while the figma design is not. Please add the grey background present in the the rounded corners
- Kindly reduce the padding applied to your modal buttons and make sure the button icons are showing